### PR TITLE
feature (refs T30295): allow to configure external links section

### PR DESCRIPTION
**Pull request by dresslerdemos**
**Ticket:** https://yaits.demos-deutschland.de/T30295

Allows to define links to external sites, that can be used by the FE to show them in the UI.

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Logic/DefaultTwigVariablesService.php [hunk 1]
```diff
@@ -215,6 +215,7 @@ public function loadVariables(Request $request): void
             'urlPathPrefix'                    => $this->globalConfig->getUrlPathPrefix(),
             'urlScheme'                        => $this->globalConfig->getUrlScheme() ?? $request->getScheme(),
             'useOpenGeoDb'                     => $this->globalConfig->getUseOpenGeoDb(),
+            'main_page_external_links'         => $this->globalConfig->getMainPageExternalLinks(),
```
Comment (graupnerdemos) [inline demosplan/DemosPlanCoreBundle/Logic/DefaultTwigVariablesService.php:218]: Please stick to the naming scheme.

**Review by graupnerdemos — COMMENTED** [review]
Technically looks good. One style mishap though.

File: demosplan/DemosPlanCoreBundle/Logic/DefaultTwigVariablesService.php (hunk 1 above)
Comment (dresslerdemos) [reply demosplan/DemosPlanCoreBundle/Logic/DefaultTwigVariablesService.php:218]: done in 0f36bf8d50b0b713dab6ca1a4d26bff6531af83a

**Review by graupnerdemos — APPROVED** [review]
lgtm

**Comment by mussbach** [comment]
sonarcloud check did pass but somehow was not updated here. Will force merge

**Pull request merged**